cli/src/cast.rs

```diff
@@ -680,6 +680,25 @@ async fn main() -> eyre::Result<()> {
             generate(shell, &mut Opts::command(), "cast", &mut std::io::stdout())
         }
         Subcommands::Run(cmd) => cmd.run()?,
+        Subcommands::Request { rpc_url, method, params } => {
```

Comment:
Do want to call this `cast rpc` instead to match [`seth rpc`](https://github.com/dapphub/dapptools/blob/master/src/seth/libexec/seth/seth-rpc)?

Reply:
ah nice! haha I was debating which to call it actually but I didn't realize the matching here...